frontend/src/app/updatesweb/create-modify.component.ts

```diff
@@ -749,6 +740,42 @@ export class CreateModifyComponent implements OnInit, OnDestroy {
         });
     }
 
+    private calculateStatusForInetnumInUpdate() {
+        const statusAttr: IAttributeModel = this.whoisResourcesService.getSingleAttributeOnName(this.attributes, 'status');
+        if (statusAttr.value === 'ALLOCATED PA') {
+            this.optionList.status = [{ key: 'ALLOCATED-ASSIGNED PA', value: 'ALLOCATED-ASSIGNED PA' }];
+            return;
+        }
+
+        if (statusAttr.value === 'ALLOCATED-ASSIGNED PA') {
+            this.optionList.status = [{ key: 'ALLOCATED PA', value: 'ALLOCATED PA' }];
```

Comment:
This logic should only be for modify not create. Are we checking this ?

Reply:
yes, check the name of the method `calculateStatusForInetnumInUpdate` which is only being called by `fetchDataForModify`  which is when this.operation = this.MODIFY_OPERATION;